bedrock/mozorg/urls.py

```diff
@@ -146,7 +146,6 @@
     ),
     path("antiharassment-tool/", views.anti_harassment_tool_view, name="mozorg.antiharassment-tool"),
     page("rise25/nominate/", "mozorg/rise25/landing.html"),
-    page("rise25/thanks/", "mozorg/rise25/thanks.html"),
```

Comment:
**question:** Is it worth adding a redirect for this URL to avoid a 404 from somewhere?

Reply:
Maybe a 404 is the best response in this case? It was only intended as the landing after submitting a nomination, and all those forms are deactivated. It didn't have any real content and shouldn't have been linked or indexed anywhere else.